docs/Using-Fleet/Fleet-desktop.md

```diff
@@ -16,6 +16,9 @@ At its core, Fleet Desktop gives your end users visibility into the security pos
    <iframe src="https://www.youtube.com/embed/cI2vDG3PbVo" allowfullscreen></iframe>
 </div>
 
+## OS Support
```

Comment:
Agh sorry I just clicked approve and then noticed this title needs sentence-casing:
```suggestion
## OS support
```